méli-mélo/2021-05-conjunction/index.html

```diff
@@ -100,10 +102,79 @@ <h4>Always stacked (mobile first, no side-by-side)</h4>
 	&lt;li class=&quot;cnjnctn-col&quot;&gt;
 		This is content for part B
 	&lt;/li&gt;
-&lt;/li&gt;</code>
+&lt;/li&gt;
+&lt;/ul&gt;</code>
   </pre>
   </details>
 
+<h3>No border on always stacked <span class="label label-info">New</span></h3>
+<p>This can be used when a border or design (e.g. zebra striping) has already defined the border of the content.</p>
+<h4>Always stacked (mobile first, no side-by-side)</h4>
+<div class="panel panel-default">
+    <header class="panel-heading">
+      <h5 class="panel-title">Heading</h5>
+    </header>
+    <div class="panel-body">
+  <ul class="cnjnctn-type-or brdr-0">
+<li class="cnjnctn-col">This is content for option A </li>
+    <li class="cnjnctn-col">This is content for option B</li>
+          </ul>
+	</div>
+</div>
+ <div class="panel panel-default">
+    <header class="panel-heading">
+      <h5 class="panel-title">Heading</h5>
+    </header>
+    <div class="panel-body">
+        <ul class="cnjnctn-type-and brdr-0">
+    <li class="cnjnctn-col">This is content for part A </li>
+    <li class="cnjnctn-col">This is content for part B</li>
+</ul>
+	 </div>
+</div>
+
+   <details>
+    <summary>
+            <p>Code</p>
```

Comment:
Summary element can only contain [phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content) and optionally intermixed with [heading content](https://html.spec.whatwg.org/multipage/dom.html#heading-content). (reference: https://html.spec.whatwg.org/multipage/interactive-elements.html#the-summary-element )

The element "p" is a [flow content](https://html.spec.whatwg.org/multipage/dom.html#flow-content) not a [phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content)

Propose change:

```suggestion
            Code
```